src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int FIGURE_COUNT = 100;
+    private Ball ball = new Ball();
```

Comment:
lottery should create new ball each time `getRandomBall` method is being called, so put this line into the method
```suggestion
```